doc/source/tutorial/formats.rst

```diff
@@ -31,9 +31,9 @@ specifies all entry types in a given dataset::
     11   Woldemort   Владимир      vladimir    Russian      6
     12   Woldemort   Володимир     volodimir   Ukrainian    6
     9    Harry       Harald        haralt      German       7
-    10   Harry       Harry         hæri        English      7
-    11   Harry       Гарри         gari        Russian      7
-    12   Harry       Гаррi         hari        Ukrainian    7
+    10   Harry       Harry         hæri        English      8
```

Comment:
I mean: Woldemart is also only obliquely cognate to "Waldemar", right? And the German books have of course also "Woldemort" for translation. But otherwise, we would not have the fun cases to test the alignments, even if they are not used in the documentation (for Harry, I mean). 

BTW: it is extremely difficult to find actual cases from linguistics where you can show all aspects of alignments...